### PR TITLE
project: jsign project first version

### PR DESCRIPTION
**Pull request by jsign**
This PR contains the first version of my project.

### CODE REVIEW

File: projects/verkle-trie-migration-exploration.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- Detailed public documents (e.g., HackMD docs) that will do an in-depth analysis of details about the problem to be solved. This helps document and onboard more people into current affairs and challenges.
+- Take a data-driven approach to validate hypotheses and bottlenecks. I’ll create reproducible benchmarks available in open-source repos that anyone can verify and reproduce. The idea is that all the work can keep evolving and isn’t lost in some code snippet.
+- Open PRs into relevant repos to help with needed work (e.g: [go-ethereum](https://github.com/ethereum/go-ethereum), [go-verkle](https://github.com/gballet/go-verkle), [go-ipa](https://github.com/crate-crypto/go-ipa))
+- Open discussion with mentors in #verkle-trie-migration so we can have a history of discussions and help raise awareness of some of the current work with Verkle Tries.
+
+## Roadmap
+
+Since there’s a lot of work happening quite fast in this area, it’s a bit hard to have a meaningful and stable roadmap; but I’d imagine the following steps:
+
+- Gain an understanding of Verkle Tries specification and new cryptographic primitives.
```
Comment (taxmeifyoucan) [inline projects/verkle-trie-migration-exploration.md:47]: You can also make these check boxes for continues tracking :)

**Review by taxmeifyoucan — APPROVED** [review]
Thanks for submitting the project!

File: projects/verkle-trie-migration-exploration.md (hunk 1 above)
Comment (jsign) [reply projects/verkle-trie-migration-exploration.md]: Nice, I've updated these to todo-bullets. Thanks for the idea!

**Comment by jimmygchen** [comment]
@jsign nice write up, this is inspiring - I like your approach here and looking forward to see more of your work!

**Pull request merged**